scene/main/node.cpp

```diff
@@ -1947,6 +1947,19 @@ HashMap<NodePath, int> Node::get_editable_instances() const {
 	return data.editable_instances;
 }
 
+String Node::to_string() {
+	if (get_script_instance()) {
+		bool valid;
+		String ret = get_script_instance()->to_string(&valid);
+		if (valid) {
+			return ret;
+		}
+	}
+
+	// Only print a space if the name is valid.
+	return (get_name() ? String(get_name()) + " " : "") + "[" + get_class() + ":" + itos(get_instance_id()) + "]";
```

Comment:
The latter part is `Object::to_string()`, so you can use that directly.

Reply:
yeah good point, fixed.